mod/01_rating/set_pattern/02_metric_type_plus_right.rb

```diff
@@ -0,0 +1,23 @@
+@@options = {
+  junction_only: true,
+  assigns_type: true,
+  anchor_parts_count: 2
+}
+
+def label name
+  %{All "+#{name.to_name.tag}" cards on metric cards of type "#{name.to_name.left_name}"}
+end
+
+def prototype_args anchor
+  { name: "+#{anchor.tag}", supercard: Card.new( name: '*dummy', type: Card::MetricID, '+*metric_type'=>"[[#{anchor.trunk}]]" ) }
```

Comment:
Line is too long. [129/80]<br>Surrounding space missing for operator `=>`.<br>Space inside parentheses detected.